components/IngredientsList.js

```diff
@@ -0,0 +1,142 @@
+import ingredients from "@/assets/ingredients.json";
+import styled from "styled-components";
+
+const IngredientsList = () => {
+  const flavorColors = {
+    Bitter: "#D2691E",
+    Creamy: "#FFFACD",
+    Fresh: "#98FB98",
+    Herbal: "#228B22",
+    Pungent: "#FF6347",
+    Salty: "#F0E68C",
+    Savory: "#D2B48C",
+    Spicy: "#cf2210",
+    Sour: "#d9df0c",
+    Sweet: "#FFB6C1",
+    Tangy: "#FFA07A",
+    Umami: "#188989",
+  };
+  return (
+    <>
+      <Title>Ingredients Overview</Title>
+      <Container>
+        <List>
+          {ingredients.map((ingredient) => (
+            <ListItem
+              key={ingredient._id}
+              $color={flavorColors[ingredient.flavor]}
+            >
+              <ImageWrapper>
+                <StyledImage imgUrl={ingredient.imgUrl} />
+              </ImageWrapper>
+              <StyledContent>
+                <Name>{ingredient.name}</Name>
+                <br></br>
+                <Flavor>{"#" + ingredient.flavor}</Flavor>
```

Comment:
You can write it like this as well:

```suggestion
                <Flavor>#{ingredient.flavor}</Flavor>
```